### ISSUE TITLE
Can't open Logi Options+ app more than once

### ISSUE THREAD

**Issue by s-bourdon** [jcbastosportela/logioptspp#16]
Hi!

Thank you very much for this great program; it's really useful!

The only problem I have found is that once **logioptspp** is runnning, **Logi Options+** can only be opened once.  On subsequent tries, a loading icon keeps spinning forever, and **Logi Options+** never opens.  The only way to access it is to restart the PC.

Are you experiencing the same thing?

Thank you for your support!

**Comment by jcbastosportela** [comment]
Yes, but I don't think it is related to this. I experienced that on my work laptop were I can't even run this.

**Comment by s-bourdon** [comment]
Thanks for your reply!

Unfortunately, that's not what I'm experiencing.
But I think I may have found a solution!

After starting **logioptspp**, if I stop and restart **logioptionsplus_agent.exe**, **Logi Options+** can be opened as it should, indefinitely.

Would it be possible to automatically restart **logioptionsplus_agent.exe** after starting **logioptspp**?

Thank you very much for your time!
